src/components/tabs/AppTabs.vue

```diff
@@ -22,7 +22,8 @@
             flex
             items-center
             justify-center
-            hover:font-bold
```

Comment:
I'm very happy to see the bold hover going :tada:.

I had to add in extra elements here to make the bold hover work without layout problems so that can now also be removed. Everything from line 12 to 32 can now probably just be something like (not tested):

```vue
<span class="text-body-80 hover:text-body">
  {{ item.label }}
  <span v-if="item.count !== undefined" class="ml-1">
    ({{ item.count }})
  </span>
</span>
```

Reply:
Fixed here: https://github.com/beabee-communityrm/beabee-frontend/pull/245/commits/49c3c6dedce172a28ad040ba81e8f4b3f0eb4800#diff-d9f58ec032801d1531ea86aa8c89542c7c332fb516447d51b08b70bdb852fdeb